linux/aliases.bsh

```diff
@@ -15,7 +15,7 @@ fi
 #
 # .. file:: aliases.bsh
 #
-# Since bash script do not use ``alias`` by default, common external programs are soft coded using an all caps variable name (e.g. ``tar`` is called using ``"${TAR}"``. Simply source :file:`aliases.bsh` and use the variables.
+# Because bash script do not use ``alias`` by default, common external programs are soft-coded using an all caps variable name (e.g. ``tar`` is called using ``"${TAR}"``. Simply source :file:`aliases.bsh` and use the variables, instead of hard-coding "tar" everywhere, use the variable ``TAR"`` so that when the need comes, it is easier to switch which executable gets called. When an alias variable is used, it should always be surrounded by quotes, e.g. ``"${TAR}"``.
```

Comment:
Because bash scripts do not use
all-caps variable (I'm pretty sure this should be hyphenated)
For example, instead of hard-coding 
This variable should always be surrounded by quotes